### PR TITLE
Bug 1137273 - Test Case - Add imagecompare email draft save test case

### PR DESCRIPTION
**Pull request by nojunpark**

### CODE REVIEW

File: tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_setup_and_send_receive_imap_email.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+from gaiatest.apps.email.app import SetupEmail
+
+
+class TestSetupAndSendIMAPEmail(GaiaImageCompareTestCase):
+    def setUp(self):
+
+        if not GaiaTestEnvironment(self.testvars).email.get('imap'):
+            raise SkipTest('IMAP account details not present in test variables.')
+        if not GaiaTestEnvironment(self.testvars).email.get('smtp'):
+            raise SkipTest('IMAP account details not present in test variables.')
```
Comment (davehunt) [inline tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_setup_and_send_receive_imap_email.py]: Nit: Should be 'SMTP account...'

File: tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_setup_and_send_receive_imap_email.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+from marionette_driver import Wait
+
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest import GaiaTestEnvironment
+from gaiatest.apps.email.app import Email
+from gaiatest.apps.email.app import SetupEmail
+
+
+class TestSetupAndSendIMAPEmail(GaiaImageCompareTestCase):
+    def setUp(self):
```
Comment (davehunt) [inline tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_setup_and_send_receive_imap_email.py]: Nit: Add empty line before, remove empty line after.

**Comment by npark-mozilla** [comment]
The Try test isn't being triggered it seems even after close/reopen of pull request.  Can it be triggered in some another way?

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=61a3b4cd865f7562e70319bf5b22c7821b5bf095

**Pull request merged**